### PR TITLE
feat: logzio destination

### PR DESCRIPTION
**Pull request by yotamloe**
### Proposed Change
Addition of logz.io destination for logs metrics and traces
- Feature request issue: #78 
- ObservIQ-otel-collector pr for logzio support: https://github.com/observIQ/observiq-otel-collector/pull/554
##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

**Comment by jsirianni** [comment]
Hi @yotamloe, thank you for this contribution. Before I run CI, can you resolve the following issues?

```
--- FAIL: TestValidateDestinationTypes (0.00s)
    --- FAIL: TestValidateDestinationTypes/destination-types/logzio.yaml (0.00s)
        resources_test.go:82: 
            	Error Trace:	/home/jsirianni/git/bindplane-op.logzio/resources/resources_test.go:82
            	Error:      	Received unexpected error:
            	            	3 errors occurred:
            	            		* template: logs.exporters:1:6: executing "logs.exporters" at <eq .enable_logs>: error calling eq: missing argument for comparison
            	            		* template: metrics.exporters:1:6: executing "metrics.exporters" at <eq .enable_metrics>: error calling eq: missing argument for comparison
            	            		* template: traces.exporters:1:6: executing "traces.exporters" at <eq .enable_tracing>: error calling eq: missing argument for comparison
            	            	
            	Test:       	TestValidateDestinationTypes/destination-types/logzio.yaml
FAIL
FAIL	github.com/observiq/bindplane-op/resources	0.021s
FAIL
```

For boolean conditions, you can omit the `eq`
```
{{ if .enable_metrics }}
```

These tests can be run with `go test ./resources` or `make test`.

**Comment by yotamloe** [comment]
Hey @jsirianni thanks for the quick reply, I'll fix this issue

File: resources/destination-types/logzio.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      default: true
+
+    - name: enable_tracing
+      label: Enable Tracing
+      description: Enable to send spans to Logz.io.
+      type: bool
+      default: true
+
+    - name: logs_token
+      label: Logs_token
```
Comment (jsirianni) [inline resources/destination-types/logzio.yaml:28]: Label names can have spaces
```suggestion
      label: Logs Token
```

**Review by jsirianni — CHANGES_REQUESTED** [review]
Giving this a spin now. Some initial thoughts:

Can you re-order the parameters so that each token comes after the enable toggle that matches it? I think this will look nicer in the UI.

Screenshot from 2022-07-26 13-35-30

**Review by jsirianni — APPROVED** [review]
Thanks!

**Review by andykellr — APPROVED** [review]

**Pull request merged**